app/models/donation.rb

```diff
@@ -1,6 +1,8 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 class Donation < ActiveRecord::Base
 
+	before_save :set_anonymous
+
```

Comment:
This isn't needed. If we deploy the change to to set the default to false, there's no time when a donation will be created with anonymous as nil.